tests/templates/kuttl/resources/10-deploy-spark-app.yaml.j2

```diff
@@ -5,9 +5,6 @@ metadata:
   name: resources-crd
 spec:
   version: "1.0"
-{% if lookup('env', 'VECTOR_AGGREGATOR') %}
-  vectorAggregatorConfigMapName: vector-aggregator-discovery
-{% endif %}
```

Comment:
Why did you remove logging to vector? (we try to log to log.t2.stackable.tech vector for all testcases to better debug tests)

Reply:
Ah, I haven't thought of that. I wanted to have cleaner tests.

Reply:
I would prefer to keep logging to e.g. collect the exceptions that occurred when the test failed